src/mame/dec/microvax3100.cpp

```diff
@@ -0,0 +1,172 @@
+  |   Xtal       |           |   ______________                                    ___________        ______________ |
+  | 14.7456 MHz  |___________|  |PAL20L10ACNS_|                                   |MC74F521N_|       | PLS105ANJ   | |
+  | ____________  ____________   ____________  __________  __________  __________  ___________       |_____________| |
+  ||SN74LS244N_| |SN74LS244N_|  |SN74LS166AN| |74F245PC_| |SN74LS139| |_74F74PC_| |74HCT245N_|        ______________ |
+  | ____________  ____________   ____________  __________  __________              ___________       | PLS105ANJ   | |
+  ||SN74LS244N_| |SN74LS244N_|  |SN74LS166AN| |74HCT245N| |22738-01_|  40MHz Xtal |_74F543N__|       |_____________| |
+  |        _____                               __________  __________  __________  ___________        ______________ |
+  |       |    |      ____                    |74HCT245N| |SN74LS375N |74F32NQST| |74HCT245N_|       | PLS105ANJ   | |
+  |       | C  |     |   |                     __________  __________  __________  ___________       |_____________| |
+  |       | O  |     | C |                    |74F245PC_| |74F32NQST| |74F32NQST| |_74F543N__|        ______________ |
+  |       | N  |     | O |                     __________  __________  __________  ___________       | EPROM       | |
+  |       | N  |     | N |                    |_74F244N_| |SN74LS09N| |74F00NQST| |74HCT245N_|       |_____________| |
+  |       |    |     | N |                     __________  __________  __________  ___________        ___________    |
+  |       |    |     |   |                    |MC74F240N| |74F32NQST| |_________| |_74F543N__|       |_74LS244N_|    |
+  |       |    |     |   |                                                         ___________        ___________    |
+  |       |    |     |___|       ______________________    ______________________ |74HCT245N_|       |_74F374N__|    |
+  |       |____|                | :::::::::::::::::::: |  | ::::::::::::::::::: |  ___________                       |
+  |                                                                               |_74F543N__|                       |
+  |__________________________________________________________________________________________________________________|
+
+***********************************************************************************************************************/
+
+#include "emu.h"
+#include "cpu/t11/t11.h"
+
+#include "machine/am79c90.h"
+//#include "machine/ncr5380.h"
+//#include "machine/scnxx562.h"
+#include "machine/terminal.h"
+
+
+namespace {
+
+class microvax3100_state : public driver_device
+{
+public:
+	microvax3100_state(const machine_config &mconfig, device_type type, const char *tag)
+		: driver_device(mconfig, type, tag)
+		, m_maincpu(*this, "maincpu")
+		, m_terminal(*this, "terminal")
+	{ }
+
+	void microvax3100(machine_config &config);
+
+private:
+	required_device<t11_device> m_maincpu;
```

Comment:
Doesn’t this need a VAX CPU, not PDP11?

Reply:
Do we have a VAX CPU core? I've only seen the disassembler...